app/frontend/src/components/Pill/Pill.tsx

```diff
@@ -18,13 +19,13 @@ export interface PillProps {
   variant?: "rounded" | "square";
 }
 
-export default function Pill({ children, variant = "rounded" }: PillProps) {
+export default function Pill({ children, variant = "square" }: PillProps) {
   const classes = useStyles();
 
   return (
     <Typography
       className={classNames(classes.root, {
-        [classes.rounded]: variant === "rounded",
+        [classes.rounded]: variant === "square",
```

Comment:
Shouldn't this be left as it is before? This is now a bit misleading/wrong as it is saying the `rounded` styling should be applied when the variant prop has value `square`.

Reply:
The issue calls for the buttons to become square. Think I need to change the classes.rounded to something else but I'm not sure what I'm looking through the codebase.

Reply:
Ok I'm not sure where exactly you mean. Thanks in regards to the root. I should not have updated that yarn package haha. Is this the part your talking about changing the variant prop in?
`export interface PillProps {
  children: React.ReactNode;
  variant?: "rounded" | "square";
}`

Reply:
Do you mean I ought to change the variant props in ReferenceListItem.tsx?

Reply:
Ok, that fixed it!

Reply:
Is there anything else I need to change here? I'm not on the Slack at the moment and can't assign anyone for a review.